YourToDo/forms.py

```diff
@@ -0,0 +1,41 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.contrib.auth.forms import UserCreationForm
+
+class UniqueUserEmailField(forms.EmailField):
+    """
+    An EmailField which only is valid if no User has that email.
+    """
+    def validate(self, value):
+        super(forms.EmailField, self).validate(value)
+        try:
+            User.objects.get(email = value)
+            raise forms.ValidationError("The specified email already belongs to a registered user")
+        except User.MultipleObjectsReturned:
+            raise forms.ValidationError("The specified email already belongs to a registered user")
+        except User.DoesNotExist:
+            pass
+
+class defaultUserRegistrationForm(UserCreationForm):
+	username = forms.CharField(label = 'Username', max_length=30, help_text=('Length: 30 characters or fewer. Letters, digits and '
```

Comment:
We might want to change password requirements in the future, to account for short passwords. This will do for now.